src/mbgl/gl/context.cpp

```diff
@@ -276,13 +276,14 @@ bool Context::supportsProgramBinaries() const {
         return false;
     }
 
-    // Blacklist Adreno 3xx, 4xx, and 5xx GPUs due to known bugs:
+    // Blacklist Adreno 2xx, 3xx, 4xx, and 5xx GPUs due to known bugs:
     // https://bugs.chromium.org/p/chromium/issues/detail?id=510637
     // https://chromium.googlesource.com/chromium/src/gpu/+/master/config/gpu_driver_bug_list.json#2316
     // Blacklist Vivante GC4000 due to bugs when linking loaded programs:
     // https://github.com/mapbox/mapbox-gl-native/issues/10704
     const std::string renderer = reinterpret_cast<const char*>(glGetString(GL_RENDERER));
-    if (renderer.find("Adreno (TM) 3") != std::string::npos
+    if (renderer.find("Adreno (TM) 2") != std::string::npos
```

Comment:
This blacklists program binaries, not VAO. The check for VAO blacklisting is a few lines further up ;)

Reply:
🤦‍♂️  should be in order with https://github.com/mapbox/mapbox-gl-native/pull/11047/commits/42633a3085d3181ec8d76a222f110589ff59b005